src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass_LocalFunctions.cs

```diff
@@ -101,6 +104,8 @@ protected TLocalFunctionState GetOrCreateLocalFuncUsages(LocalFunctionSymbol loc
                 PendingBranches.Add(new PendingBranch(null, this.State, null));
             }
 
+            VisitAttributes(localFuncSymbol);
```

Comment:
Why is this the right location to call 'VisitAttributes'? Is it important that we run this after 'EnterParameters', for example?